jcl/src/jdk.management/share/classes/com/ibm/lang/management/internal/PlatformMBeanProvider.java

```diff
@@ -90,6 +91,10 @@
 			.addInterface(JvmCpuMonitorMXBean.class)
 			.register(allComponents);
 
+		ComponentBuilder.create(OpenJ9DiagnosticsMXBeanImpl.getInstance())
```

Comment:
This class is only used for Java 9. com.ibm.java.lang.management.internal.ManagementUtils needs to be updated for Java 8.